src/main/java/at/esque/kafka/PublisherController.java

```diff
@@ -126,7 +126,8 @@ private void setupControls(ObservableList<Integer> partitions, TopicMessageTypeC
                 }
             }
         }
-        if (MessageType.AVRO_TOPIC_RECORD_NAME_STRATEGY.equals(configForTopic.getValueType())) {
+        if (MessageType.AVRO_TOPIC_RECORD_NAME_STRATEGY.equals(configForTopic.getValueType())
+                || MessageType.PROTOBUF_SR.equals(configForTopic.getValueType())) {
```

Comment:
enabling this does not seem to be necessary, since the default subjectNameStrategy for the protobuf serializer is same as for AVRO, meaning that there can only be one schema for the value and one schema for the key per topic.

It will also not have any effect, because it is not used in the createProtobufMessage Method in the ProducerHandler

Reply:
Ah, okay. I've removed it. Thanks!